content/guides/ecommerce-website.md

```diff
@@ -0,0 +1,29 @@
+---
+title: eCommerce 
+description: How to redirect users back to an e-commerce website after paying/cancelling a Centrapay Payment Request.
+nav:
+  path: Reference/Merchant Integrations
+  order: 12
+---
+
+eCommerce websites accepting payments via Centrapay may want to redirect their users back to their website after they pay/cancel the payment request. In order to do this, the eCommerce website must add `allowedRedirectUrls` to it’s [Merchant Config](https://docs.centrapay.com/api/merchant-configs).
+
+After `allowedRedirectUrls` have been added to the Merchant Config, the eCommerce website can simply pass in their `redirectUrl` when creating a [Payment Request](https://docs.centrapay.com/api/payment-requests).
+
+POST `/api/payment-requests`
```

Comment:
This section looks really strange on the website.

Reply:
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/50732795/217719746-e5f41bf7-910f-450f-b573-fe24d60bfb89.png">

Reply:
Could you please elaborate on this? Thanks. 🙏

Reply:
Megan came and helped to elaborate :)
![Screenshot 2023-02-10 at 11 33 04 AM](https://user-images.githubusercontent.com/32785419/217954409-f32eaf17-2441-46f4-bb44-459b0855d85d.png)

Reply:
Nice! Looks good!